dcpandas/setup.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Install dcpandas."""
```

Comment:
Can you add more description here? Are clients supposed to run this, or is this more to set up the pip library (I think the latter bu tI'm unclear).

Reply:
Also what's the difference with the other setup file below?

Reply:
I changed the description to """Build and distrubute the datacommons_pandas package to PyPI."""

This one is for the pandas wrapper library, the other is for the regular datacommons